### ISSUE TITLE
Editor not accepting uploadConfig prop

### ISSUE THREAD

**Issue by ahammednafih** [neetozone/neeto-editor#924]
The storybook mentions that the `ImageUpload` default configs could be overwritten by passing the options to the [uploadConfig](https://neeto-editor.neeto.com/?path=/story/welcome--page&hash=eaebcefd50a62437db13c21a33b83fa34fa75136) prop. But the editor doesn't seem to be accepting that prop.

discussion _t
@AbhayVAshokan _a

**Comment by AbhayVAshokan** [comment]
@ahammednafih we removed any overrides after making sure we need the same requirements everywhere. Documentation changes were missed out. We have the values hardcoded inside the frontend now. Do we need to make it flexible for any use cases?

**Comment by ahammednafih** [comment]
@AbhayVAshokan We are standardizing the file size limits with https://github.com/bigbinary/neeto-engineering-web/issues/419. As part of this change, wanted to send the configured file size available via `globalProps.endUserUploadedFileSizeLimitInMb` to neetoEditor. This is the use-case.

**Comment by AbhayVAshokan** [comment]
@ahammednafih in that case would it be better to directly import `globalProps.endUserUploadedFileSizeLimitInMb` within neetoEditor codebase rather than passing it as a prop?

**Comment by ahammednafih** [comment]
@AbhayVAshokan That would work as well. Since we already hv it hardcoded on the codebase, I don't think there will be a custom use case.

**Comment by ahammednafih** [comment]
cc: @sandip-mane

**Comment by sandip-mane** [comment]
For this, we can import it directly in the neetoEditor so that we don't need to pass it from everywhere we use the editor.
@AbhayVAshokan could you assign it to someone?

**Comment by AbhayVAshokan** [comment]
I'll take this up. @AbhayVAshokan _a

**Comment by AbhayVAshokan** [comment]
@ahammednafih one more query. neetoEditor also supports uploading videos and attachments (both set to 100MB as of now). Should this apply to them as well?

**Comment by ahammednafih** [comment]
For attachments, this could be applied(we have `attachmentConfig` to override it?). @sandip-mane Could you comment on the video size part?

**Comment by sandip-mane** [comment]
@AbhayVAshokan let's keep the video size to 100mb. We will modify that in a separate issue if needed. (We need to know whats the average size for uploaded videos, this will need some data before we can implement something)

**Comment by AbhayVAshokan** [comment]
> For attachments, this could be applied(we have attachmentConfig to override it?)

I shall update the default value to `globalProps. endUserUploadedFileSizeLimitInMb` within neetoEditor so that this need not be edited everywhere.
